components/brave_rewards/browser/CHANGELOG.md

```diff
@@ -21,4 +21,18 @@ to determinate which country user is in.
 |  Return value |    | `true` or `false`        |
 |  Parameters   |    |                          |
 
----
\ No newline at end of file
+---
+### Refactoring (9th October 2019)
+##### Related PR
+
+[https://github.com/brave/brave-core/pull/3630](https://github.com/brave/brave-core/pull/3630)
+
+##### Description
+
+Refactored and migrated Rewards category from `category` to `type`.
+
+##### Change
+
+Renamed `RewardsCategory` to `RewardsType` and added migration path.
```

Comment:
please add note that this also effect JS return data so UI needs to check if they are using `category` and change to `type`

Reply:
Fixed